src/main/scala/chisel3/Driver.scala

```diff
@@ -95,6 +95,20 @@ object Driver extends BackendCompilationUtilities {
 
   def emit[T <: RawModule](ir: Circuit): String = Emitter.emit(ir)
 
+  def emitVerilog[T <: RawModule](gen: () => T): String = {
```

Comment:
Why are two signatures needed? And can you just make one call the other (so we can be a little more DRY)?